_schemaV2.graphql

```diff
@@ -7241,32 +7430,32 @@ enum MarketingGroupTypes {
 # Market Price Insights
 type MarketPriceInsights {
   annualLotsSold: Int
-  annualValueSoldCents: BigInt
+  annualValueSoldCents: AnalyticsBigInt
```

Comment:
Are we sure these type diffs (all the types that now have an `Analytics` prefix) won't break existing clients that are querying MarketPriceInsights? Two places I'd think might be affected --- 

1. Non-update mobile apps. Will they have any issues if the underlying types returned by the queries are named differently?
2. Any existing typescript that is referencing individual types from these queries. I don't know if any exist, but if they do, they would need to change. Is that a better outcome than updating our stitching code to keep the type names all the same?